lib/trino-filesystem/src/test/java/io/trino/filesystem/AbstractTestTrinoFileSystem.java

```diff
@@ -880,7 +881,7 @@ public void testDirectoryExists()
             }
             else {
                 assertThat(getFileSystem().directoryExists(createLocation(directoryName))).isEmpty();
-                createBlob(closer, createLocation(directoryName).appendPath(fileName).path());
+                createBlob(closer, "%s/%s".formatted(directoryName, fileName));
```

Comment:
Should line 877 be changed as well? i.e. for the equivalent `createBlob` for hierarchical filesystems.